daemon/openrazer_daemon/dbus_services/dbus_methods/mamba.py

```diff
@@ -166,6 +166,48 @@ def get_dpi_xy(self):
     return dpi
 
 
+@endpoint('razer.device.dpi', 'setDPIStages', in_sig='aq')
+def set_dpi_stages(self, dpi_stages):
+    """
+    Set the DPI on the mouse, Takes in pairs of 2 bytes big-endian
+
+    :param dpi_stages: pairs of dpi X and dpi Y for each stage
+    :type dpi_stages: list of int
+    """
+    self.logger.debug("DBus call set_dpi_stages")
+
+    driver_path = self.get_driver_path('dpi_stages')
+
+    dpi_bytes = b''
+    for dpi in dpi_stages:
+        dpi_bytes += struct.pack('>H', dpi)
+
+    with open(driver_path, 'wb') as driver_file:
+        driver_file.write(dpi_bytes)
+
+
+@endpoint('razer.device.dpi', 'getDPIStages', out_sig='aq')
```

Comment:
I'm not sure if the data format for `get/setDPIStages` is the best idea. Right now they receive/return a list of ushort that is parsed as:
```
[stage1_dpi_x, stage1_dpi_y, stage2_dpi_x, stage2_dpi_y, stage3_dpi_x, stage3_dpi_y, ...]
```

Alternatives would be:
* Receive/return a list of lists: `[[stage1_dpi_x, stage1_dpi_y], [stage2_dpi_x, stage2_dpi_y], [stage3_dpi_x, stage3_dpi_y], ...]`
* Receive 2 lists: one for dpi X values and one for dpi Y values. Return a flatten list like it does now or a list of lists like in the previous option.

Reply:
A list of tuples would be good I think? dbus signature should be `a(qq)` I believe

Reply:
Done in 349c09d8dd9dc28676590aed51dc3ef5e5fb41ae